handbook/product/README.md

```diff
@@ -113,10 +113,14 @@ If a bug is unreleased or [critical](https://fleetdm.com/handbook/engineering#cr
 
 If a bug is not addressed within 6 weeks, [it is sent to the product team for triage](https://fleetdm.com/handbook/engineering#in-engineering). During the sprint kickoff review ritual, each product group PM should review every bug which is marked in this way with the Head of Product and recommended whether a bug should be kept for the next sprint or de-prioritized. A de-prioritized bugs might include action items such as documentation of the defect. The Head of Product then reviews the bug recommendations with the CEO in the bug de-prioritization review ritual. 
```

Comment:
I think this should be updated, right? We're no longer doing it at the sprint review kickoff ritual. It's being done separately.

Reply:
@zhumo I've updated to capture process changes.